setup.py

```diff
@@ -41,7 +41,7 @@
 setup(
     name="cadquery",
     use_scm_version=True,
-    # version="2.3.0",  # Uncomment this for the next release
+    version="2.3.0dev",  # Update this for the next release
```

Comment:
Does this override the automated SCM version?

Reply:
There was a suggestion in [this comment] (https://github.com/CadQuery/cadquery/pull/1085#issuecomment-1137708449) on how it might be possible to get the automatic SCM versioning working with Conda too.

Reply:
For the release conda package I get the correct version. When installing from a local repo, I get 0.0.0. #1085 already mentions issues, so it is maybe better to just remove it.

Reply:
Does moving this line out of the if statement fix the SCM version issue?
https://github.com/CadQuery/cadquery/blob/2d355177871c0b1a275640659ba203cf3cdf267c/setup.py#L39

Reply:
It does work with this addition and BTW `use_scm_version` does override `version`. I'd like to still get rid of all this magic and just report the version as 2.3dev. Do you see any practical added value in not removing it?

Reply:
> Do you see any practical added value in not removing it?

No, not really.  The simpler `2.3dev` should be good enough.

Reply:
TBH I'd really like to get rid of this. If it is needed `git reflog` can be used. So far, I have never asked for exact commit. When reporting issues, you should reproduce first on the current master.

Reply:
Sounds reasonable, thanks for taking the time to explain!